Rakefile

```diff
@@ -89,6 +90,32 @@ def generate_ember_data_docs
   puts "Built #{repo_path} with SHA #{sha}"
 end
 
+def geocode_meetups
+  data_path = 'meetups.yml'
+  puts "Geocoding records from #{data_path}... "
+
+  data = YAML.load_file(File.expand_path("./data/#{data_path}"))
+  data["locations"].each do |loc|
+    loc["groups"].each do |group|
+      next if group.has_key?("lat") and group.has_key?("lat")
+      coord = Geocoder.coordinates(group["address"]) || Geocoder.coordinates(group["location"])
+      if coord.nil?
+        puts "Unable to find coordinates for #{group["location"]}"
+        next
+      end
+      group["lat"] = coord[0]
+      group["lng"] = coord[1]
+      puts "Found coordinates for #{group["location"]}"
+      #throttle requests to API to avoid errors
```

Comment:
Probably a good idea, and I guess it could also provide a place in the code for data preparation using the meetups.com API that has been discussed for the future. I will attempt a refactoring later - hopefully this evening